eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/Examples.hs

```diff
@@ -387,18 +387,18 @@ exampleProposedPPUpdates =
       (mkKeyHash 1)
       (emptyPParamsUpdate & ppuMaxBHSizeL .~ SJust 4000)
 
-examplePoolParams :: PoolParams
-examplePoolParams =
-  PoolParams
-    { ppId = hashKey $ vKey $ mkDSIGNKeyPair 1
-    , ppVrf = exampleVrfVerKeyHash
-    , ppPledge = Coin 1
-    , ppCost = Coin 5
-    , ppMargin = unsafeBoundRational 0.1
-    , ppRewardAccount = exampleRewardAccount
-    , ppOwners = Set.singleton $ hashKey $ vKey exampleStakeKey
-    , ppRelays = StrictSeq.empty
-    , ppMetadata =
+exampleStakePoolParams :: StakePoolParams
```

Comment:
This is an exported binding. We need to mention this rename in the changelog under `### testlib` section